CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 Change Log
 ==========
 
+## 1.17.0-beta01
+- Synchronizer APIs for listing sent and received transactions have been removed
```

Comment:
It would probably be helpful to describe here what users of these APIs should migrate to.

Reply:
This is covered in the MIGRATIONS.md file